common.mk

```diff
@@ -233,6 +236,7 @@ $(TOP_SMEMS_CONF) $(MODEL_SMEMS_CONF) &:  $(MFC_SMEMS_CONF) $(MFC_MODEL_HRCHY_JS
 		--model-module-name $(MODEL) \
 		--out-dut-smems-conf $(TOP_SMEMS_CONF) \
 		--out-model-smems-conf $(MODEL_SMEMS_CONF)
+	cat $(base_dir)/vlsi/add.mems.conf >> $(TOP_SMEMS_CONF)
```

Comment:
What is this for?

Reply:
ah i had a comment here, somehow missed it and didnt push. updated, it's for the srams we use inside of vortex